cfgov/unprocessed/js/modules/util/dom-traverse.js

```diff
@@ -0,0 +1,69 @@
+'use strict';
+
+require( '../polyfill/class-list' );
+
+/**
+ * Get the sibling nodes of a dom node.
+ *
+ * @param {HTMLNode} elem - DOM element to get siblings of.
+ * @param {string} className - CSS classname.
+ * @returns {Array} List of sibling nodes.
+ */
+function getSiblings( elem, className ) {
+  var siblings = [];
+  var sibling = elem.parentNode.firstChild;
+  for ( ; sibling; sibling = sibling.nextSibling ) {
+    // Check that sibling is an HTMLNode,
+    // is not the original element,
+    // and contains the CSS class we're looking for.
+    if ( sibling.nodeType === 1 &&
+         sibling !== elem &&
+         sibling.classList.contains( className ) ) {
+      siblings[siblings.length] = sibling;
+    }
+  }
+  return siblings;
+}
+
+/**
+ * Get the sibling nodes of a dom node.
```

Comment:
Fixed in https://github.com/cfpb/cfgov-refresh/commit/ba60774032060677129040d4f3bb7742fc05171c